scripts/build-glean-metadata.py

```diff
@@ -62,12 +65,27 @@ def _serialize_sets(obj):
                 "description": metric.description,
             }
         )
+
+        stable_ping_table_names = []
+        for ping in metric.definition["send_in_pings"]:
+            stable_ping_table_names.append(
+                [ping, f"{app_id_snakecase}.{stringcase.snakecase(ping)}"]
+            )
+
+        metric_type = metric.definition["type"]
+        metric_name_snakecase = stringcase.snakecase(metric.identifier)
+        bigquery_names = dict(
+            stable_ping_table_names=stable_ping_table_names,
+            metric_table_name=f"metrics.{metric_type}.{metric_name_snakecase}",
+        )
+
         open(os.path.join(app_metrics_dir, f"{metric.identifier}.json"), "w").write(
             json.dumps(
                 dict(
                     metric.definition,
                     name=metric.identifier,
```

Comment:
Should probably serialize the `metric_type` here, so we can easily access it later (see below)